### PR TITLE
Remove dead links from addonFiles

### PR DESCRIPTION
**Pull request by seanbudd**
Removes dead links:
* tmast
* imgdesc
* bgt

### CODE REVIEW

File: get.php [hunk 1]
```diff
@@ -63,10 +62,8 @@
 	"fh-dev" => "https://github.com/nvdajp/focusHighlight/releases/download/6.6/focusHighlight-6.6.nvda-addon",
 	"gc" => "https://github.com/nvda-es/goldenCursor/releases/download/6.1.1/goldenCursor-6.1.1.nvda-addon",
 	"gc-dev" => "https://github.com/nvda-es/goldenCursor/releases/download/6.1.1/goldenCursor-6.1.1.nvda-addon",
-	"gl" => "https://github.com/nvdaaddons/nvdaaddons.github.io/wiki/guideLines",
```
Comment (seanbudd) [inline get.php:66]: Perhaps this should not be removed, as it still may be used.

**Comment by josephsl** [comment]
Hi, I advise keeping “gl” for use by the add-ons community Thanks.

File: get.php [hunk 2]
```diff
@@ -66,7 +65,6 @@
 	"gl" => "https://github.com/nvdaaddons/nvdaaddons.github.io/wiki/guideLines",
```
Comment (seanbudd) [inline get.php:65]: note to self: how should this be resolved when migrating to the legacy view? append manually?

**Comment by josephsl** [comment]
Hi,

Keep in mind that comunity add-ons website do include links to dead links to be removed in this PR. I'll inform the add-ons community about it.

Tanks.

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**